switchboard/dds_util.py

```diff
@@ -168,12 +168,18 @@ def __init__(self, delivery_or_share, user):
         self.delivery = delivery_or_share
         self.ddsutil = DDSUtil(user)
         self.user = user
+        self._from_user = None
+        self._to_user = None
 
     def get_from_user(self):
```

Comment:
Compared to `s3_util` (see below), these make sense as lazily-loaded, and one could argue they'd work better as `@property`s too.